src/pages/AccessProfilePage/Modal/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { Button } from 'antd';
+import { boolean } from 'yup';
+
+import { roles } from '../MockedData';
+import { ModalComponent, CascaderComponent, InputComponent, InputWrapper, Label } from './styles';
+
+interface IProps {
```

Comment:
É interessante nós criamos nomenclaturas melhores pra interface, assim, se ela for compartilhável fica fácil de identificar o que é o que.
Ex: ao invés de "IProps" , poderia ser "ModalModel" ou "AccessProfileModalModel"